lms/static/scripts/frontend_apps/components/FilePickerApp.tsx

```diff
@@ -480,7 +485,23 @@ export default function FilePickerApp({ onSubmit }: FilePickerAppProps) {
                       <>
                         <div className="sm:col-span-2 border-b" />
                         <PanelLabel isCurrentStep verticalAlign="center">
-                          Max points
+                          <div className="flex items-center justify-end">
+                            Max points
+                            <IconButton
+                              icon={InfoIcon}
+                              title="About max points"
+                              onClick={() =>
+                                setMaxPointsPopoverOpen(prev => !prev)
+                              }
+                              expanded={maxPointsPopoverOpen}
+                              elementRef={iconRef}
+                              // Add right negative margin that compensates for
+                              // the button padding.
+                              // That way the button surface is the same, but
+                              // it looks aligned with the other elements
```

Comment:
I didn't immediately understand what this meant. The intent here is that the right edge of the "info" icon aligns with the right edge of the text labels above and below. To achieve this, we have to compensate for the button's internal padding. This static margin is correct when using a mouse, but the button gets larger when using touch input:

<img width="502" alt="Misaligned button" src="https://github.com/user-attachments/assets/1947ccce-88d1-4196-b4d9-d1706a2e1337" />